doc/markdown/user/implementation/library/Usage.md

```diff
@@ -8,15 +8,15 @@ Accelerator Executable Functions
 
 Functions that should be executable on an accelerator have to be annotated with the execution domain (one of `ALPAKA_FN_HOST`, `ALPAKA_FN_ACC` and `ALPAKA_FN_HOST_ACC`).
 They most probably also require access to the accelerator data and methods, such as indices and extents as well as functions to allocate shared memory and to synchronize all threads within a block. 
-Therefore the accelerator has to be passed in as a constant template reference variable as can be seen in the following code snippet.
+Therefore the accelerator has to be passed in as a templated constant reference parameter as can be seen in the following code snippet.
```

Comment:
I'm not against it, just saying that I've probably never seen "templated" used in such constructs (e.g. also on line 34). On the other hand, it is a short way to express that.

Reply:
Ah, ofc I've never seen it, as "template" is only a noun, not adjective or verb, so "templated" makes no grammatical sense. Again, we might still keep it as it is not likely to cause confusion.

Reply:
https://www.linguee.de/englisch-deutsch/uebersetzung/to+be+templated.html

Reply:
I've read templated as adjective many times before, it's slang and ok if you ask me: https://www.google.com/search?q=google+"templated"+"C%2B%2B"